packages/builders/src/util/normalizeArray.ts

```diff
@@ -0,0 +1,6 @@
+export function normalizeArray<T>(arr: RestOrArray<T>): T[] {
+	if (Array.isArray(arr[0])) return arr[0];
+	return arr as T[];
+}
+
+export type RestOrArray<T> = T[] | T[][];
```

Comment:
```suggestion
export type RestOrArray<T> = T[] | [T[]];
```

As you can see [here](https://www.typescriptlang.org/play?#code/C4TwDgpgBAShDOwDyAnAgilBDEAeAKgHxQC8U+A2gLpQA+UFlVVAsAFDsAmEAxgDZYU0AGYBXAHY9gASwD24qFk6cAwrIC2YeRHHB4ACgB0xnhq3idegFywEydJhy5xo9QCMIKQgEobEgNbisgDu4uzsSqpm2roGAIwANFAATEkAzN4RymqaMXr6FIkp6VSZbJE55pYGhUmpUGlUSbXFDaXhbKCQtoioGNggyQTEZEx05NTUHdz8giISUnIKFdEWsclGJqvVNnC9DgNDLu6ePn7igSFhHOXZ2+v6RfUZWVG5a3obLc-tt29VD2+JWaTxK3iAA), the third example will only fail with the updated type.